src/commands/cmd_tdigest.cc

```diff
@@ -38,7 +38,6 @@ constexpr auto kInfoUnmergedWeight = "Unmerged weight";
 constexpr auto kInfoObservations = "Observations";
 constexpr auto kInfoTotalCompressions = "Total compressions";
 }  // namespace
-
```

Comment:
Hi @SharonIV0x86 ,

Could you help keep this blank line for namespace section? 😊

Best Regards,
Edward